doc/script_commands.txt

```diff
@@ -7961,17 +7961,15 @@ Return values:
 //
 ---------------------------------------
 
-*npcspeed <speed value>;
-*npcwalkto <x>,<y>;
-*npcstop;
+*npcspeed( <speed value> {,"<npc name>"} );
+*npcwalkto( <x>,<y> {,"<speed value>" {,"<npc name>"} } );
+*npcstop( {"<npc name>"} );
 
-These commands will make the NPC object in question move around the map. As they
-currently are, they are a bit buggy and are not useful for much more than making
-an NPC move randomly around the map.
+These commands will make the NPC object in question move around the map.
 
 'npcspeed' will set the NPCs walking speed to a specified value. As in the
-@speed GM command, 200 is the slowest possible speed while 0 is the fastest
-possible (instant motion). 100 is the default character walking speed.
+@speed GM command, 1000 is the slowest possible speed while 20 is the fastest
+possible (instant motion). 200 is the default npc walking speed.
```

Comment:
Should we export the constants and use them here?
And should we create a constant for the default walk npc walkspeed?